### ISSUE TITLE
Got error when train data

### ISSUE THREAD

**Issue by danielnguyen1612** [pbcquoc/crnn#5]
Chào bạn, mình đang theo hướng dẫn của bạn để train với dataset `InkData_line_processed` nhưng khi mình chạy thì gặp lỗi 
![image](https://user-images.githubusercontent.com/1705651/76546984-042fe400-64bf-11ea-8c2b-c515ddf3cc77.png)
Không biết bạn có thể hướng dẫn mình để resolve lỗi này không ? TIA

**Comment by swelouis** [comment]
Bạn ơi, đổi height đầu vào là 32 nha :)))

**Comment by blueclowd** [comment]
Yeah, i think we need to modify the network for other shape. However, according to the readme, we can train directly with height=64?
